class = class classname(){
}

constructor = classname(args){
}

input from user = Scanner sc = new Scanner()

nextInt= integer and nextLine= String

try{
code to be tested for exception}

catch(exception e){
system.out.println("something went wrong");}

finally{
system.out.println(anything u want to say)}

try {
			int[] mynumber = {1,2,3};
			System.out.println(mynumber[10]);
		}
		catch (Exception e) {
			System.out.println("Something went wrong");
		}
		finally {
			System.out.println("The 'try catch' is finished.");
		}
